mne_bids/mne_bids.py

```diff
@@ -586,12 +632,19 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         raise ValueError('ext must be in %s, got %s'
                          % (''.join(ALLOWED_EXTENSIONS), ext))
 
-    # for FIF, we need to re-save the file to fix the file pointer
-    # for files with multiple parts
+    # Copy the imaging data files
+    # Re-save FIF files to fix the file pointer for files with multiple parts
```

Comment:
we should a comment that this is not currently implemented. I am working on it with this pr: https://github.com/mne-tools/mne-python/pull/5470

Reply:
@teonbrooks I guess we would need a `copyfile_fif` function eventually? Or is your MNE PR going to take care of it?